app/views/manage/events/_form.html.haml

```diff
@@ -0,0 +1,10 @@
+= bs_horizontal_simple_form_for @event, url: url_for(action: @event.new_record? ? "create" : "update", controller: "events") do |f|
+  .col-8.mx-auto
```

Comment:
I think this looks really off when the rest of our forms aren't adapted yet. I say we keep it the standard form design until we do a sweep of the whole app for a UI sync/refresh.

Reply:
I agree